src/data/veiledere/veilederTypes.ts

```diff
@@ -1,10 +1,11 @@
-export interface VeilederInfoDto {
+export interface VeilederDTO {
   ident: string;
-  navn?: string;
+  fornavn: string;
+  etternavn: string;
+  epost: string;
+  telefonnummer?: string;
 }
 
-export interface VeilederDto {
-  ident: string;
-  fornavn?: string;
-  etternavn?: string;
+export function fullName(veileder: VeilederDTO): string {
```

Comment:
Kunne godt returnert full navn som en del av veilederQuery-hooken.

Reply:
Hvordan tenker du da? Som en del av responsen? Eller i selve hooken? 😊

Reply:
Går an å bruke `select` for å legge på et navn på dataene som returneres fra hooken:
```
return useQuery({
    queryKey: veilederQueryKeys.veileder,
    queryFn: fetchVeileder,
    select: (veileder) => {
      return { ...veileder, navn: veileder.fornavn + " " + veileder.etternavn };
    },
  });
```
Da slipper vi også endre alle steder som bruker navn i dag. Eller returnere det som et eget felt/funksjon fra hooken i tillegg til useQuery. :) Synes i hvert fall det er fint å få det fra hooken på et vis.